log4j-core/src/test/java/org/apache/logging/log4j/core/appender/rolling/RollingAppenderDirectWriteTempCompressedFilePatternTest.java

```diff
@@ -96,8 +97,7 @@ public void testAppender() throws Exception {
                 }
             }
             assertTrue("No temporary file created during compression", temporaryFilesCreated > 0);
-            assertTrue("Temporarys file created not equals to compressed files",
-                    compressedFiles == temporaryFilesCreated);
+            assertEquals("Temporarys file created not equals to compressed files", compressedFiles, temporaryFilesCreated);
```

Comment:
I wish this would be a comparison against a `Set<Path>` of files. Would you make this change, please?